drivers/SCsub

```diff
@@ -23,7 +23,7 @@ SConscript('coremidi/SCsub')
 SConscript('winmidi/SCsub')
 
 # Graphics drivers
-if (env["platform"] != "server"):
+if (env["vulkan_enabled"]):
```

Comment:
This if is intended to disable ALL renderers for the headless builds, not just Vulkan one.